docs/conf.py

```diff
@@ -61,52 +37,29 @@
 version = package_info.__version__
 release = package_info.__version__
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['sdk_index.rst']
+# exclude_patterns = ['']
 
 linkcheck_retries = 2
 linkcheck_anchors = False
 linkcheck_ignore = [r'https://cloud.dwavesys.com/leap',  # redirects, many checks
                     ]
+language = "en"
 
-# The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
-
 # -- Options for HTML output ----------------------------------------------
+html_theme = 'pydata_sphinx_theme'
+html_theme_options = {
+    "collapse_navigation": True,
+    "show_prev_next": False,
+}
+html_sidebars = {"**": ["search-field", "sidebar-nav-bs"]}  # remove ads
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-import sphinx_rtd_theme
-html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-html_static_path = ['_static']
-
-def setup(app):
-    app.add_css_file('cookie_notice.css')
-    app.add_js_file('cookie_notice.js')
-    app.add_config_value('target', 'repo', 'env')
```

Comment:
cookie notice is not needed anymore? or is that handled at the sdk level only?

Reply:
we no longer publish to RTD site for cloud client package, where this was used in the past